Resources/Public/Javascript/PageView/SearchInDocument.js

```diff
@@ -15,9 +15,9 @@
   * @returns void
   */
 function nextResultPage() {
-    var currentstart = $("#tx-dlf-search-in-document-form input[name='tx_dlf[start]']").val();
+    var currentstart = $("#tx-dlf-search-in-document-form input[id='tx-dlf-search-in-document-start]']").val();
```

Comment:
```suggestion
    var currentstart = $("#tx-dlf-search-in-document-form input[id='tx-dlf-search-in-document-start']").val();
```
All other lines are wrong, too. This is only a sample suggest.